profiles-sdk-examples/Makefile

```diff
@@ -11,28 +11,39 @@ SPARK_CONTAINER := ${DOCKER_PREGISTRY_URL}/${DOCKER_IMAGE}:${TAG}
 create-app-image:
 	docker build --build-arg base_img=c12e/spark-template:profile-jar-base-6.3.0-M.2 -t profiles-example:latest -f ./main-app/build/resources/main/Dockerfile ./main-app/build
 
-build: clean
+# Build the Application
+build:
 	./gradlew build
 
+# Test the Application
+test:
+	./gradlew test
+
 clean:
 	./gradlew clean
 
 
+# Tag the latest create-app-image built container
 tag-container: check-env
 	docker tag profiles-example:latest ${SPARK_CONTAINER}
 
+# Push the container
 push-container: check-env
 	docker push ${SPARK_CONTAINER}
 
+# Deploy the action and save the skill
```

Comment:
can we have something like `make all: clean build create-app-image deploy-skill invoke`, in case I want to blindly point to the cluster(`cortex configure set-profile dev`) and run the example. @laguirre-cs @mcriscolo-cs

Reply:
Having `make all` sounds like one honking great idea! Though there is no "blindly pointing" to a cluster because the a user will always need to update the spark-submit config file to match (which is built into the application image).

I've thought about allowing optional config overrides in the skill payload, so a user wouldn't have to rebuild/push the image for a config file update, but hadn't gotten around to that